src/02-map.js

```diff
@@ -13,7 +13,10 @@ const exampleSongData = require("../data/songs");
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {string[]} An array of strings, all of which are song titles.
  */
-function getSongTitles(songs) {}
+function getSongTitles(songs) {
+  const songTitles = songs.map((song) => song.title);
+  return songTitles;
```

Comment:
another way to write this is to simply return what the function returns. 
`return songs.map((song) => song.title);`

The same goes for the rest of them.